hoot-core/src/main/cpp/hoot/core/io/OsmGbdxXmlWriter.cpp

```diff
@@ -521,21 +518,13 @@ void OsmGbdxXmlWriter::_writeWayWithPoints(const ConstWayPtr& w, ConstOsmMapPtr
 
   _writer->writeEndElement(); // WKT
 
-//  _writer->writeStartElement("type");
-//  _writer->writeCharacters(featureGeometry);
-//  _writer->writeEndElement();
-
```

Comment:
Removed some old commented out code in some of the files I modified, while I was in there.